src/es_archiver/lib/docs/generate_doc_records_stream.js

```diff
@@ -54,7 +54,7 @@ export function createGenerateDocRecordsStream(client, stats) {
             this.push({
               type: 'doc',
               value: {
-                index: hit._index,
+                index: hit._index.startsWith('.kibana') ? '.kibana_1' : hit._index,
```

Comment:
This makes sense to me and to you. I wonder if a comment would help others here, though, as I suspect it's a confusing line otherwise.